recipes/shtns/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "shtns" %}
+{% set version = "3.3.1" %}
+{% set revision = "r694" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://www.bitbucket.org/nschaeff/{{ name }}/downloads/{{ name }}-{{ version }}-{{ revision }}.tar.gz
+    sha256: 3fcca0ffcf28467fd80ef71e0032c15ac6deac5c592041dc6a9a7f4575fbe64c
+    folder: src
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+
+  build:
+    - m2-bash  # [win]
+    - m2-curl  # [win]
+    - m2-findutils  # [win]
+    - m2-diffutils  # [win]
+    - m2-gawk  # [win]
+    - m2-grep  # [win]
+    - m2-make  # [win]
+    - m2-make  # [win]
+    - m2-sed  # [win]
+    - m2-which  # [win]
+    - m2-tar  # [win]
+    - swig  # [win]
+    - {{ compiler('c') }}  # [unix]
+    - {{ compiler('m2w64_c') }}  # [win]
+  host:
+    - fftw
+    - m2w64-binutils  # [win]
+    - m2w64-gcc  # [win]
+    - numpy >=1.12
+    - python
+  run:
+    - fftw
+    - {{ compiler('c') }}
```

Comment:
Do you need a compiler at run time?

Reply:
And this compiler seems inconsistent on windows

Reply:
Re: the windows compiler, I've removed the windows specific build sections per the dropping of windows support.

Re: the compiler at runtime, I believe I might have originally put that in the requirements due to the dependency on libm (`-lm`), but that's now been removed.